pytorch/src/main/resources/org/bytedeco/pytorch/presets/torch_include.h

```diff
@@ -1398,18 +1410,15 @@
 #include "torch/csrc/jit/frontend/error_report.h"
 #include "torch/csrc/jit/frontend/tree.h"
 #include "torch/csrc/jit/frontend/lexer.h"
+#include "caffe2/serialize/inline_container.h"
+#include "torch/csrc/jit/serialization/import.h"
+#include "c10/util/FbcodeMaps.h"
+#include "torch/csrc/jit/serialization/pickler.h"
 #include "torch/csrc/jit/frontend/parser_constants.h"
 #include "torch/csrc/jit/frontend/strtod.h"
 #include "torch/csrc/jit/frontend/schema_matching.h"
 #include "torch/csrc/jit/frontend/versioned_symbols.h"
 #include "torch/csrc/jit/frontend/tree_views.h"
-#include "torch/csrc/jit/serialization/pickler.h"
+#include "torch/csrc/jit/serialization/pickle.h"
 
-// Parsed and for inclusion in JNI
-// See also https://github.com/pytorch/pytorch/blob/main/docs/cpp/source/Doxyfile
-// for an approximation of what should be in API in addition to torch.h"
-// torch/csrc/jit/runtime/custom_operator.h:  Name conflict with torch::RegisterOperator + little chance to have any use
-#include "torch/torch.h"
-#include "ATen/native/TensorShape.h"
-#include "torch/csrc/jit/serialization/storage_context.h"
-#include "torch/csrc/jit/serialization/import.h"
```

Comment:
What's the reason we don't need those anymore?

Reply:
`torch.h` is the main root include that draws others. It only contains includes. We can add it for good measure, but this won't change anything.

It's unclear what exactly is part of the "public" C++ API besides `torch.h`. In 2.0.1 I mainly considered what is in the doxygen files that generates the online api doc. But then I realized that the normal, documented, way to access the TorchScript functionality is to include `script.h`. So in 2.1 what is parsed is what is drawn by `torch.h` + `script.h`. That's closer to how the lib is to be used.
`import.h` is drawn by `script.h` and is parsed. `TensorShape.h` and `storage_context.h` are not. I guess they are not meant to be accessed by users.